diff --git a/tools/git/gitdiff.sh b/tools/git/gitdiff.sh
index ade1f6a..66c544e 100755
--- a/tools/git/gitdiff.sh
+++ b/tools/git/gitdiff.sh
@@ -8,7 +8,7 @@ mkdir -p tmp
 COMMIT_ID=$(git rev-parse --short HEAD)
 
 # 获取当前时间戳，格式为：YYYYMMDD_HHMM (例如：20260311_0945)
-TIMESTAMP=$(date +"%Y%m%d_%H%M")
+TIMESTAMP=$(date +"%Y%m%d%H%M")
 
 # 定义文件名
 FILENAME="gitdiff_tmp/temp_diff_${TIMESTAMP}_${COMMIT_ID}.diff"
